utils/validators.ts

```diff
@@ -14,7 +14,7 @@ import xecaddr from 'xecaddrjs'
 /* Validates the address and adds a prefix to it, if it does not have it already.
  */
 export const parseAddress = function (addressString: string): string {
-  if (addressString === '') throw new Error(RESPONSE_MESSAGES.ADDRESS_NOT_PROVIDED_400.message)
+  if (addressString === '' || addressString === undefined) throw new Error(RESPONSE_MESSAGES.ADDRESS_NOT_PROVIDED_400.message)
```

Comment:
Since this check is being made, shouldn't the typing for the function be `... = function (addressString: string | undefined)` ?